rtCamp/next-pms
# feat: add the confirmation dialog for allocation deletion

> Please provide enough information so that others can review your pull request:



- Added the confirmation dialog for allocation deletion.

> Explain the **details** for making this change. What existing problem does the pull request solve?



- Added a custom component `DeleteAllocation` as a confirmation dialog using shadcn `Dialog`.

Why not use react-alert-dialog?
- The above package was creating issues for form dynamic field selection, and it has bugs in recent versions: https://github.com/shadcn-ui/ui/issues/1655 so for now I have used dialog only.

> QA List



- Build the PMS pages `cd /apps/next_pms/frontend/pms && npm run build`
- `bench restart`
- Check if the dialog is visible on resource deletion or not

> Screenshots/GIFs



https://github.com/user-attachments/assets/d56e7117-db5d-42f1-ac0d-f0c1ce42137a
